packages/xstate-vue/README.md

```diff
@@ -61,9 +61,9 @@ A [Vue composition function](https://vue-composition-api-rfc.netlify.com/) that
 - `machine` - An [XState machine](https://xstate.js.org/docs/guides/machines.html).
 - `options` (optional) - [Interpreter options](https://xstate.js.org/docs/guides/interpretation.html#options) OR one of the following Machine Config options: `guards`, `actions`, `activities`, `services`, `delays`, `immediate`, `context`, or `state`.
 
-**Returns** `{ current, send, service}`:
+**Returns** `{ state, send, service}`:
```

Comment:
in `@xstate/react` we return a tuple instead of an object, was a decision about returning an object here based on Vue conventions or maybe was it done by accident? should we change this to a tuple?

Reply:
This is based on Vue composition API, where you always return an object, so I think no, we shouldn't change that to a tuple.